src/prefect/agent/agent.py

```diff
@@ -131,8 +128,9 @@ def __init__(
         agent_address: str = None,
         no_cloud_logs: bool = None,
     ) -> None:
-        # Load token and initialize client
+        # Load token for backwards compatibility
```

Comment:
what about backwards compatibility for config.cloud.agent.api_key?

Reply:
`config.cloud.agent.api_key` was never a thing

Reply:
ahhh gotcha, got confused since you can set api keys under config.cloud.agent.auth_token